src/Converters/Converter.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace GrahamCampbell\Markdown\Converters;
+
+use GrahamCampbell\Markdown\Responses\MarkdownString;
+
+class Converter extends \League\CommonMark\Converter {
+	
+	/**
+	 * Converts CommonMark to HTML.
+	 *
+	 * @param string $commonMark
+	 *
+	 * @return string
+	 *
+	 * @api
```

Comment:
Please don't use the API annotation.

Reply:
That block was autogenerated by Eclipse. Completely looked over that, sorry.
Adressed in 44be4b8316b891964e4c04726fd4bd0d6bd5c2e9